scripts/delay_xrfi_run.py

```diff
@@ -42,4 +38,11 @@
 io.update_uvdata(dfil.input_data, data=dfil.filtered_residuals, flags=dfil.flags)
```

Comment:
I tried `delay_xrfi_run.py zen.2458098.12552.xx.HH.uv` and got error below:

  File "/users/thuang/lustre/hera_qm/scripts/delay_xrfi_run.py", line 38, in <module>
    io.update_uvdata(dfil.input_data, data=dfil.filtered_residuals, flags=dfil.flags)
AttributeError: Delay_Filter instance has no attribute 'input_data'